internal/common/baseclient/baseSpeaker.go

```diff
@@ -11,11 +12,17 @@ type BaseSpeaker struct {
 func (s *BaseSpeaker) PayJudge() shared.SpeakerReturnContent {
 	return shared.SpeakerReturnContent{
 		ContentType: shared.SpeakerJudgeSalary,
-		JudgeSalary: 10,
+		JudgeSalary: s.GetJudgeSalary(),
 		ActionTaken: true,
 	}
 }
 
+// GetJudgeSalary gets the Judge's salary from the rules.
+func (s *BaseSpeaker) GetJudgeSalary() shared.Resources {
+	JudgeSalaryRule := rules.AvailableRules["salary_cycle_judge"]
+	return shared.Resources(JudgeSalaryRule.ApplicableMatrix.At(0, 1))
```

Comment:
i think if this is to be a utility function it should check rulesinplay and return a struct of something like {value int, rulepresent bool}